css/DateRangePicker.scss

```diff
@@ -61,13 +68,19 @@
   border-bottom-color: rgba(0, 0, 0, 0.1);
 }
 
-.DateRangePicker__picker--start::before {
+.DateRangePicker__picker--direction-left.DateRangePicker__picker--start::before {
   left: 22px;
 }
+.DateRangePicker__picker--direction-right.DateRangePicker__picker--start::before {
+  right: 239px;
+}
 
-.DateRangePicker__picker--end::before {
+.DateRangePicker__picker--direction-left.DateRangePicker__picker--end::before {
   left: 162px;
 }
+.DateRangePicker__picker--direction-right.DateRangePicker__picker--end::before {
+  right: 101px;
```

Comment:
unsure if there's a measurement strategy that should be used to calculate these offsets, just tried to eye them. Let me know if there's a calculation you were relying on (and the we can add a `$sass-variable`)